pkg/apis/gke.cattle.io/v1/types.go

```diff
@@ -33,105 +33,300 @@ type GKEClusterConfig struct {
 
 // GKEClusterConfigSpec is the spec for a GKEClusterConfig resource
 type GKEClusterConfigSpec struct {
-	Region                         string                             `json:"region" norman:"noupdate"`
-	Zone                           string                             `json:"zone" norman:"noupdate"`
-	Imported                       bool                               `json:"imported" norman:"noupdate"`
-	Description                    string                             `json:"description"`
-	Labels                         map[string]string                  `json:"labels"`
-	EnableKubernetesAlpha          *bool                              `json:"enableKubernetesAlpha"`
-	ClusterAddons                  *GKEClusterAddons                  `json:"clusterAddons"`
-	ClusterIpv4CidrBlock           *string                            `json:"clusterIpv4Cidr" norman:"pointer"`
-	ProjectID                      string                             `json:"projectID"`
-	GoogleCredentialSecret         string                             `json:"googleCredentialSecret"`
-	ClusterName                    string                             `json:"clusterName"`
-	KubernetesVersion              *string                            `json:"kubernetesVersion" norman:"pointer"`
-	LoggingService                 *string                            `json:"loggingService" norman:"pointer"`
-	MonitoringService              *string                            `json:"monitoringService" norman:"pointer"`
-	NodePools                      []GKENodePoolConfig                `json:"nodePools"`
-	Network                        *string                            `json:"network,omitempty" norman:"pointer"`
-	Subnetwork                     *string                            `json:"subnetwork,omitempty" norman:"pointer"`
-	NetworkPolicyEnabled           *bool                              `json:"networkPolicyEnabled,omitempty"`
-	PrivateClusterConfig           *GKEPrivateClusterConfig           `json:"privateClusterConfig,omitempty"`
-	IPAllocationPolicy             *GKEIPAllocationPolicy             `json:"ipAllocationPolicy,omitempty"`
+	// +optional
```

Comment:
Normally the kubebuilder validation comments come at the end of the comment block

Reply:
Thanks, done